config/license-whitelist.yml

```diff
@@ -0,0 +1,19 @@
+Apache 2.0
```

Comment:
There's a good chance `config` will never contain any other files and a whitelist isn't configuration anyway strictly speak, wdyt about just putting this file in the repo root? Would require the action to be updated with the correct path.